repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/media/MediaRangeProcessor.java

```diff
@@ -52,18 +52,4 @@ public MediaType process(HeaderValue headerValue) {
 
         return new MediaType(mediaTypeWithParametersStripped, mediaType, headerValue.getParameters());
     }
-
-    public static List<MimeType> getMimeTypesFromHeaderValues(List<String> headerValues) {
```

Comment:
This was only being used by the rate limiting filter. This felt like an overly specific method that was also in the wrong spot, so I modified it a bit and moved it into `MimeType`.